src/theory/arith/kinds

```diff
@@ -80,6 +80,13 @@ operator LEQ 2 "less than or equal, x <= y"
 operator GT 2 "greater than, x > y"
 operator GEQ 2 "greater than or equal, x >= y"
 
+constant INDEXED_ROOT_PREDICATE_OP \
+	::CVC4::IndexedRootPredicate \
+	::CVC4::IndexedRootPredicateHashFunction \
+	"util/arith.h" \
+	"operator for the indexed root predicate; payload is an instance of the CVC4::IndexedRootPredicate class"
```

Comment:
I'm not sure about the issue of "polynomial compared to zero".  If we allow rewriting of applications of this node, I would expect that the third parameter is an arbitrary arithmetic literal (that is in turn interpreted as a polynomial compared to zero).

Reply:
Ah right. I changed my mind about there I feel more comfortable having the sign condition and eventually settled for the second argument (with the variable) instead of the polynomial, but forgot to update the documentation.

Reply:
I see, ok.